packages/near-fast-auth-signer-e2e-tests/package.json

```diff
@@ -14,20 +14,24 @@
   ],
   "scripts": {
     "test": "yarn playwright test",
-    "start": "webpack serve --mode development --history-api-fallback"
+    "start": "webpack serve  --mode development --history-api-fallback"
   },
   "dependencies": {
+    "@covalenthq/client-sdk": "^1.0.2",
```

Comment:
I'm not sure if it's possible. 

But can we avoid adding test dependencies on the app dependencies? In order to avoid a big package.